runtime/include/chpl-comm.h

```diff
@@ -109,13 +109,14 @@ void chpl_comm_wait_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles);
 int chpl_comm_try_nb_some(chpl_comm_nb_handle_t* h, size_t nhandles);
 
 // Returns whether or not the passed wide address is known to be in
-// a communicable memory region - that is, a region for which it is
-// guaranteed that puts/gets will succeed without access violation or
-// other memory protection error.
+// a communicable memory region and known to be readable. That is,
+// GET to that address should succeed without an access violation
+// or other memory protection error.
 // Returns 1 if the entire passed region is known to be in the
 // registered memory region, or 0 if some or all of it is not (or
-// the communicable memory region is totally unknown).
-int chpl_comm_is_in_segment(c_nodeid_t node, void* start, size_t len);
+// the registered memory region is totally unknown as with e.g.
+// GASNET_SEGMENT=everything).
+int chpl_comm_addr_gettable(c_nodeid_t node, void* start, size_t len);
```

Comment:
Yes, but it needs to check *both* that on the remote locale the address could be read *and* that it can be read using chpl_comm_get(_nb). What if we called it chpl_comm_addr_remote_readable ?